tutorials/scripting/evaluating_expressions.rst

```diff
@@ -0,0 +1,208 @@
+
+    var expression = Expression.new()
+    expression.parse("20 + 10*2 - 5/2.0")
+    var result = expression.execute()
+    print(result)  # 37.5
+
+The following operators are available:
+
++------------------------+-------------------------------------------------------------------------------------+
+| Operator               | Notes                                                                               |
++========================+=====================================================================================+
+| Addition ``+``         | Can also be used to concatenate strings and arrays:                                 |
+|                        | - ``"hello" + " world"`` = ``hello world``                                          |
+|                        | - ``[1, 2] + [3, 4]`` = ``[1, 2, 3, 4]``                                            |
++------------------------+-------------------------------------------------------------------------------------+
+| Subtraction (``-``)    |                                                                                     |
++------------------------+-------------------------------------------------------------------------------------+
+| Multiplication (``*``) |                                                                                     |
++------------------------+-------------------------------------------------------------------------------------+
+| Division (``/``)       | Performs and integer division if both operands are integers.                        |
+|                        | If at least one of them is a floating-point number, returns a floating-point value. |
++------------------------+-------------------------------------------------------------------------------------+
+| Modulo (``%``)         | Returns the remainder of an integer division.                                       |
++------------------------+-------------------------------------------------------------------------------------+
+
+Spaces around operators are optional. Also, keep in mind the usual
+`order of operations <https://en.wikipedia.org/wiki/Order_of_operations>`__
+applies. Use parentheses to override the order of operations if needed.
+
+All the Variant types supported in Godot can be used: integers, floating-point
+numbers, strings, arrays, dictionaries, colors, vectors, …
+
+Arrays and dictionaries can be indexed like in GDScript::
+
+    # Returns 1.
+    [1, 2][0]
+
+    # Returns 3. Negative indices can be used to count from the end of the array.
+    [1, 3][-1]
+
+    # Returns "green".
+    {"favorite_color": "green"}["favorite_color"]
+
+    # All 3 lines below return 7.0 (Vector3 is floating-point).
+    Vector3(5, 6, 7)[2]
+    Vector3(5, 6, 7)["z"]
+    Vector3(5, 6, 7).z
+
+Passing variables to an expression
+----------------------------------
+
+In addition to, you can pass variables to an expression. These variables will then
```

Comment:
"In addition to" what?  Did you mean In addition to above? You can simply skip it and just go with "You can pass variables to an expression" 

We are talking about expressions here using words like "in addition to" is confusing because it's unclear you you are talking about mathematical addition or you use a word addition to mean "also"